ts/session/messages/outgoing/content/link/DeviceLinkGrantMessage.ts

```diff
@@ -0,0 +1,60 @@
+import { SignalService } from '../../../../../protobuf';
+import { DeviceLinkRequestMessage } from './DeviceLinkRequestMessage';
+import { LokiProfile } from '../../../../../types/Message';
+
+interface DeviceLinkGrantMessageParams {
+  timestamp: number;
+  identifier: string;
+
+  // pairing authorisation
+  primaryDevicePubKey: string;
+  secondaryDevicePubKey: string;
+  requestSignature: Uint8Array;
+  grantSignature: Uint8Array;
+
+  lokiProfile: LokiProfile;
+}
+
+export class DeviceLinkGrantMessage extends DeviceLinkRequestMessage {
+  private readonly displayName: string;
+  private readonly avatarPointer: string;
+  private readonly profileKey: Uint8Array;
+  private readonly grantSignature: Uint8Array | null;
```

Comment:
this should never be null in a grant message

Reply:
Fixed